package com.pp.classy.exception;

public class UsersNotFoundException extends Throwable {
}
